docs/releases/1.9.3.txt

```diff
@@ -42,3 +42,5 @@ Bugfixes
   regressed in Django 1.9 (:ticket:`26253`).
 
 * Fixed ``BoundField`` to reallow slices of subwidgets (:ticket:`26267`).
+* Handle POST to delete protected fk relationship more appropriately in admin
```

Comment:
This issue doesn't qualify for a backport -- it's not a regression nor a major crashing issue.